app/presenters/renewal_letter_presenter.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class RenewalLetterPresenter < BaseLetterPresenter
+  # Provides the full postal address for the letter.
+  def postal_address_lines
+    lines = [contact_full_name]
+    lines << operator_name unless business_type == WasteExemptionsEngine::Registration::BUSINESS_TYPES[:sole_trader]
+    lines << address_lines(contact_address)
+    lines.flatten!.reject(&:blank?)
```

Comment:
[NIT] this can also benefit from the use of the `presence` and `compact` methods

Reply:
See my previous comment about replacing `reject()` with `presence` and `compact`.